digitalforms-api/src/main/java/ca/bc/gov/open/pssg/rsbc/digitalforms/controller/PaymentStatusController.java

```diff
@@ -34,7 +32,7 @@
 	@Autowired
 	PaymentService paymentService;
 
-	@GetMapping(value = "/{noticeNumber}/payment/status")
+	@GetMapping(value = "{formType}/{noticeNumber}/payment/status")
```

Comment:
We actually don't need to include the form type here as the payment status being requested is for a 'prohibition' and not the payment for the form. We can talk about this more today. Thanks.

Reply:
removed